x-pack/plugins/endpoint/common/types.ts

```diff
@@ -44,3 +62,37 @@ export interface EndpointMetadata {
     };
   };
 }
+
+export type AlertData = Immutable<{
```

Comment:
Please add a comment (in jsdoc style) to each field.

Reply:
This data will change very soon to conform to ECS. We will document more once the format is closer to the final format. Right now we are using outdated fake data.